opentracing-spring-cloud-starter/src/test/java/io/opentracing/contrib/spring/cloud/websocket/WebsocketTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2016-2017 The OpenTracing Authors
```

Comment:
@objectiser could you please create a PR for checking license headers?

Reply:
Done #54

Reply:
thanks :+1: